### ISSUE TITLE
We should add the PerformanceRole example to our actor property

### ISSUE THREAD

**Issue by thadguidry** [schemaorg/schemaorg#1807]
https://github.com/schemaorg/schemaorg/blob/9e09f4150ca7667c9ab40d82b71c8b0c503c5055/data/examples.txt#L1825

Our actor property has no examples. :(  But there is a great example on PerformanceRole !  We should copy the example from the PerformanceRole to the actor property, to better illustrate a nice use case.

**Comment by RichardWallis** [comment]
Agree

**Comment by thadguidry** [comment]
Got pinged on this for Stale, but to iterate again that this probably could be easily fixed by someone in 5 mins.  Perhaps someone could do that and then quickly just close this issue.  All the best!

**Comment by MarieCooperActor** [comment]
It would also be useful to be able to have the option to add reviews of an actor's performance role in a play or in a movie too. There is no way to currently associate an actor with a play, as the schema seems to assume a play is a static literary piece of creative work.

**Comment by RichardWallis** [comment]
PR (#2859) raised to implement original suggestion.

@MarieCooperActor - As per the description of [Play](https://schema.org/Play):

> Note the performance of a Play would be a [TheaterEvent](https://schema.org/TheaterEvent) - the Play being the workPerformed.

This is where the relationship with an actor would be established.  An _actor_ in a _TheaterEvent_ in which the _workPerformed_ would be the _Play_ in question.

**Comment by thadguidry** [comment]
@RichardWallis This is one of those simple in life, but complex in practice, scenarios that really needs an additional JSON-LD example on TheaterEvent. Like we did with with provided example for Movies and Performances.

**Comment by chrisspradling1980** [comment]
👍

**Comment by WeaverStever** [comment]
PerformanceRole would be very handy to have available under Person > performerIn which currently only supports the Event type which is very large and unwieldy when creating a **person**'s resume (Past events require so many irrelevant parameters in SDTT).  

It would be great to have actor and musician as enumerations for performerin.  

The example on https://schema.org/PerformanceRole needs to be updated to comply with SDTT.
~~~~
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "Movie",
  "name": "Ghostbusters",
  "image": "https://en.wikipedia.org/wiki/Ghostbusters#/media/File:Ghostbusters_(1984)_theatrical_poster.png",
  "director": "Ivan Reitman",
  "dateCreated": "1984-06-08",
  "sameAs": "http://en.wikipedia.org/wiki/Ghostbusters",
  "actor": {
    "@type": "PerformanceRole",
    "actor": {
      "@type": "Person",
      "name": "Bill Murray"
    },
    "characterName": "Dr. Peter Venkman"
  }
}
</script>
~~~~

**Comment by chrisspradling1980** [comment]
👍

Also, maybe include actors to enable a preview of environment type settings
such as, a place of style preference and what may take place inside the
role itself.

**Comment by RichardWallis** [comment]
Fixed by PR #2859
